CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.25.0] - 2024-02-28
+
```

Comment:
pyincore release PR says 2/22. Maybe should be 22?

Reply:
oh is it? Let me ask in slack

Reply:
Changed to today